docs/guides/update-guide/800-to-810.md

```diff
@@ -0,0 +1,26 @@
+---
+id: 800-to-810
+title: Update 8.0 to 8.1
+description: "Review which adjustments must be made to migrate from Camunda Platform 8.0.x to Camunda Platform 8.1.0"
+---
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from Camunda Platform 8.0.x to 8.1.0 for each component of the system.
+
+## Server
+
+### Zeebe
+
+No changes are required to update Zeebe 8.0 to 8.1.
+But there are deprecated properties that you should be aware of.
+:::note
+Even if these parameters are deprecated you can still use them because of the backward compatibility of the Zeebe versioning system.
+:::
+Deprecated properties:
+
+- Zeebe Gateway's property `contactPoint`
+  - Remove the usage of `zeebe.gateway.cluster.contactPoint` property.
+  - Use the `zeebe.gateway.cluster.initialContactPoints` property. Note: you can set the list of broker addresses here, to make the start process of the Zeebe Gateway more resilient. Please, refer to [the Zeebe Gateway's configuration](../../../self-managed/zeebe-gateway-deployment/zeebe-gateway/#cluster-configuration) guide for detail.
+  - Remove the usage of `ZEEBE_GATEWAY_CLUSTER_CONTACTPOINT` environment variable.
+  - Use the `ZEEBE_GATEWAY_CLUSTER_INITIALCONTACTPOINTS` environment variable. Note: you can set the list of broker addresses here, to make the start process of the Zeebe Gateway more resilient. Please, refer to [the Zeebe Gateway's configuration](../../../self-managed/zeebe-gateway-deployment/zeebe-gateway/#cluster-configuration) guide for detail.
```

Comment:
:thought_balloon: Can we merge this somehow? Instead of repeating the content, just refer to the property and env var?